web-www/components/organisms/MarkdownSection/MarkdowmSection.tsx

```diff
@@ -1,7 +1,7 @@
 import React, { ReactNode } from 'react';
 import { Box } from '@mui/material';
-import { makeStyles } from '@mui/styles';
 import clsx from 'clsx';
+import { makeStyles } from 'tss-react/mui';
```

Comment:
Why is this change here?

Reply:
`makeStyles` from `@mui/styles'` is deprecated and should not be used anymore. We migrated all these imports to `tss-react/mui`, this is a remaining import from this migration.